test/helpers.js

```diff
@@ -0,0 +1,165 @@
+import test from 'ava';
+
+import fs from 'fs';
+import path from 'path';
+import 'babel-register';
+import 'babel-polyfill';
+import {
+  checksum,
+  getAddCard,
+  getCssTemplate,
+  getDb,
+  getLastItem,
+  getMedia,
+  getSave,
+  getSql,
+  getTemplate,
+  getZip,
+  rand
+} from '../src/helpers';
+
+test('checksum', t => {
+  t.plan(2);
```

Comment:
Please read [when-to-use-plan.md](https://github.com/avajs/ava/blob/master/docs/recipes/when-to-use-plan.md)